pkg/client/unversioned/autoscaling.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
```

Comment:
This is checked by verify-boilerplate script. We should update the script i change everywhere I guess.